Python/compile.c

```diff
@@ -2881,7 +2881,7 @@ compiler_try_except(struct compiler *c, stmt_ty s)
                   try:
                       # body
                   finally:
-                      name = None
+                      name = None # in case name is unbound during except handling
```

Comment:
How about going down to basics and just say `in case "# body" contains "del name"` ?

Reply:
Even better. Maybe leave out the `#`.